docs/setup/piweb_backend.md

```diff
@@ -17,7 +17,7 @@ The *PiWeb Cloud* is a manageable fully cloud-based backend for *PiWeb* that can
 [PiWeb Cloud FAQ](https://piwebcloud.metrology.zeiss.com/faq-en){:target="_blank"}
 
 ## PiWeb Enterprise Server
-*PiWeb Enterprise Server* is an on-premises installable *PiWeb* backend. It requires a running *Microsoft SQL Server*. Although the installation of *PiWeb Enterprise Server* is fairly easy, quite some effort is required to configure the connection to the SQL server, setup user authentication, configure user permissions, ... and many more aspects.
+*PiWeb Enterprise Server* is an on-premises installable *PiWeb* backend. It requires a running *Microsoft SQL Server*. Although the installation of *PiWeb Enterprise Server* is fairly easy, quite some effort is required to configure the connection to the SQL server, setup user authentication, configure user permissions ... and many more aspects.

```

Comment:
"... and many more aspects."
That sounds quite negative, perhaps we can rewrite it.

Reply:
Well it is certainly not meant to be positive either. Maybe we could  neutralize it a bit by saying "and some other aspects"?